app/src/marketplace/components/StreamSelector/index.jsx

```diff
@@ -97,24 +99,25 @@ export const StreamSelector = (props: Props) => {
 
     const { hasError, error } = useLastError(rest)
 
+    const isDisabled = disabled || fetchingStreams
```

Comment:
```js
const disabled = disabledProp || fetchingStreams
``` 

For some reason, I find aliasing stuff coming from props with the `Prop` suffix more readable down the road. Just a suggestion. All up to you. 🤝

Reply:
Ok, I agree that can be more readable. I'll work on the `StreamSelector` more when I add the paged loading so I'll update it there.